cpg_workflows/metamist.py

```diff
@@ -126,6 +126,17 @@
     """,
 )
 
+GET_COHORT_NAMES_QUERY = gql(
```

Comment:
can we drop this whole query and just return `id` when we query for the SGs in a cohort above. Maybe slightly rephrasing that `SGByCohortQuery` query as `GetCohortContents` to be more specific

Reply:
yeah I thought the separate query may be clear-er, but it does add an additional call we don't really need. good catch!